synapse/storage/schema/main/delta/72/04remove_unstable_private_read_receipts.sql

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2022 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+DELETE FROM receipts_linearized WHERE receipt_type = "org.matrix.msc2285.read.private";
+DELETE FROM receipts_graph WHERE receipt_type = "org.matrix.msc2285.read.private";
```

Comment:
I think you'll need single quotes:

```suggestion
DELETE FROM receipts_linearized WHERE receipt_type = 'org.matrix.msc2285.read.private';
DELETE FROM receipts_graph WHERE receipt_type = 'org.matrix.msc2285.read.private';
```